### PR TITLE
Base client for Hawkular Alerts

### PR DESCRIPTION
**Pull request by lucasponce**
@burmanm a preliminar base client for Alerting.
Following PRs will add more business method to interact with alerts.

### CODE REVIEW

File: hawkular/alerts.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+                 user,
+                 password,
+                 tenant_id,
+                 host='localhost',
+                 port=8080,
+                 path='hawkular/alerts'):
+
+        self.user = user
+        self.password = password
+        self.tenant_id = tenant_id
```
Comment (abonas) [inline hawkular/alerts.py:34]: iirc in the ruby client it is a string/tenant name. is it named here tenant_id for some other purpose/reason?

Comment (lucasponce) [inline hawkular/alerts.py:34]: In the python client, this parameter is called tenant_id.
I am not going to change it on this PR.

Comment (burmanm) [inline hawkular/alerts.py:34]: tenant_id is used in the Hawkular-Metrics codebase, so that's where it originally comes from.

**Comment by lucasponce** [comment]
This PR has low priority, as it was more a personal effort rather than something that is planned to do.
I talked with colleagues that are searching for alerting capabilities and having that under the python client will help.
Is this PR stalled for some reason ?
My plan was to add small contributions as far as I have free slots.
Do you prefer another strategy (a complete feature set before to merge) ?
Thanks.

**Comment by burmanm** [comment]
Yes, I would prefer a commit that achieves something. Adding the alerts requires changing the building system, the README structure (in PyPi also) and merging some common code. For that reason I would not like to see stubs in the master branch.

**Comment by lucasponce** [comment]
Ok

**Pull request closed without merging**